Makefile

```diff
@@ -222,15 +167,15 @@ vet: ## Run go vet against code.
 
 .PHONY: modules
 modules: ## Update Go dependencies.
-	go get $(MCAD_REPO)@$(MCAD_VERSION)
-	go get github.com/ray-project/kuberay/ray-operator
+	#go get $(MCAD_REPO)@$(MCAD_VERSION)
```

Comment:
Can we just remove this make directive?

Reply:
Good catch, thanks. It was a temporary work-around until all the pieces finally clicked. Updated in a243e589b40e1523cee6660ed83cc5f1a3690a91.